MediaSessionExt.cpp

```diff
@@ -204,15 +226,23 @@ CDMi_RESULT MediaKeySession::GetChallengeDataExt(uint8_t * challenge, uint32_t &
         passedChallenge = nullptr;
     }
 
+#ifdef NETFLIX
     err = Drm_LicenseAcq_GenerateChallenge_Netflix(m_poAppContext,
+#else
+    err = Drm_LicenseAcq_GenerateChallenge(m_poAppContext,
+#endif
                                                    RIGHTS,
                                                    sizeof(RIGHTS) / sizeof(DRM_CONST_STRING*),
                                                    nullptr,
                                                    nullptr, 0,
                                                    nullptr, nullptr,
                                                    nullptr, nullptr,
                                                    passedChallenge, &challengeSize,
+#ifdef NETFLIX
                                                    &mNounce[0], isLDL);
+#else
+                                                   nullptr);
+#endif
```

Comment:
This code shouldn't be even called in non Netflix case, should it? It should be redesigned to abstract this out but for time being let's go with ifdeffery. Just ifdef that out so it compiles for you.

Reply:
okay